app/main.py

```diff
@@ -1,2 +1,128 @@
+from typing import Any
+
+
+class Node:
+    def __init__(self, key: Any, value: Any, hash_code: int) -> None:
+        self.key = key
+        self.value = value
+        self.hash_code = hash_code
+        self.next = None
+
+
 class Dictionary:
-    pass
+    def __init__(
+            self, initial_capacity: int = 16,
+            load_factor: float = 0.75
```

Comment:
No need to make it as the `__init__` params, these params are the details of internal implementation, user should not know about this